scripts/scil_generate_sampling_scheme.py

```diff
@@ -117,13 +104,10 @@ def main():
     parser = _build_arg_parser()
     args = parser.parse_args()
 
-    caru = args.caru
-    phil = args.phil
     fsl = args.fsl
-    siemens = args.siemens
     mrtrix = args.mrtrix
 
-    if not (caru or phil or fsl or siemens or mrtrix):
+    if not (fsl or mrtrix):
```

Comment:
Really not of fan of the outfile specification. Any way we can put the extension instead of having to put --fsl or --mrtix at the bottom?

If too complicated, just say, output prefix without extension (please see --fsl or --mrtrix option below)

Reply:
I decided to leave it that way so we make sure the extension(s) are correct and be able to output mrtrix and fsl formats at the same time.